### PR TITLE
andke80 updated profile

### PR DESCRIPTION
**Pull request by Aketti**
Update on andke80's profile and associated projects

### CODE REVIEW

File: public/content/personnel/andke80/andke80.mdx [hunk 1]
```diff
@@ -5,10 +5,14 @@ image: ""
 position: [ "Research engineer" ]
 contact_info: {
     email: "anders.kettisen@liu.se",
-    address: "Kungsgatan 54, Room G5XX, Norrköping",
+    address: "Kungsgatan 54, Room G515, Norrköping",
 }
 personal_webpage: ""
-skills: []
+skills: [Programming, AI, UX]
```
Comment (alexanderbock) [inline public/content/personnel/andke80/andke80.mdx:11]:
```suggestion
skills: [ "Programming", "AI", "UX" ]
```

File: public/content/personnel/andke80/andke80.mdx [hunk 2]
```diff
@@ -5,10 +5,14 @@ image: ""
...
 ---
 
-[[TODO]]
+Anders works as a research engineer at Linköping University with focus
```
Comment (alexanderbock) [inline public/content/personnel/andke80/andke80.mdx:14]: Feel free to write it in the first person as well, if that is more comfortable

File: public/content/personnel/andke80/andke80.mdx [hunk 3]
```diff
@@ -5,10 +5,14 @@ image: ""
...
-skills: []
+skills: [Programming, AI, UX]
 ---
 
-[[TODO]]
+Anders works as a research engineer at Linköping University with focus 
+on development of applications for research projects, and on research 
+investigation and experimentation. 
+This involves a variety of work areas such as interactive 2D and 3D application
+development, image processing, 3D-reconstruction and super computing.
```
Comment (alexanderbock) [inline public/content/personnel/andke80/andke80.mdx:18]: I'd also remove all of the newlines in the text

**Review by alexanderbock — APPROVED** [review]

File: public/content/personnel/andke80/andke80.mdx (hunk 1 above)
Comment (Aketti) [reply public/content/personnel/andke80/andke80.mdx:11]: Nice catch

**Pull request merged**